<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RB_TableWidget</class>
 <widget class="QWidget" name="RB_TableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="glMain">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbTable">
     <property name="title">
      <string>GroupBoxTable</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="1" column="0">
       <widget class="RB_TableView" name="tableView"/>
      </item>
      <item row="0" column="0">
       <widget class="RB_ToolButtonBar" name="tbb" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hlRoot">
     <item>
      <widget class="QLabel" name="lblRoot">
       <property name="text">
        <string>Root</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leRoot"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbSourceModelFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RB_ToolButtonBar</class>
   <extends>QWidget</extends>
   <header>rb_toolbuttonbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RB_TableView</class>
   <extends>QTableView</extends>
   <header>rb_tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>leRoot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
